### PR TITLE
Update documentation     

### PR DESCRIPTION
**Pull request by schaefi**
Update the documentation with regards to the move
from the plain kiwi initrd to the dracut initrd system.
This is related to Issue #576

### CODE REVIEW

**Comment by schaefi** [comment]
Pretty large doc update, I think at best you review by building the docs and reading of the chapters and sub chapters of

* Building Images
* Overview

Thanks much

File: doc/source/building/build_oem_disk.rst [hunk 1]
```diff
@@ -163,30 +163,42 @@ deployment target system:
```
Comment (tomschr) [inline doc/source/building/build_oem_disk.rst]: use `:file:` for `LimeJeOS-Leap-42.3.x86_64-1.42.3.install.tar.xz`

File: doc/source/building/build_oem_disk.rst [hunk 2]
```diff
@@ -163,30 +163,42 @@ deployment target system:
...
+   LimeJeOS-Leap-42.3.append. Edit your PXE configuration
    (for example pxelinux.cfg/default) on the PXE server and add these
    parameters to the append line
 
-   Optionally the image and its md5sum can be stored on an FTP or HTTP
-   server specified via the parameters kiwiserver=IP_ADRESS and
-   kiwiservertype=HTTP_HTTPS_OR_FTP. In this case make sure to copy the
-   system image and md5 file to the correct location on the server.
-   KIWI searches the image below SERVER_ROOT/image/
+   A typical PXE append line looks like the following example
```
Comment (tomschr) [inline doc/source/building/build_oem_disk.rst]: IMHO, this sentence seems a bit unnecessary. The following line should be enough:

```
Edit your PXE configuration (for example :file:`pxelinux.cfg/default`) on the
PXE server and add these parameters to the append line, typically
looking like this:
```

File: doc/source/building/build_oem_disk.rst [hunk 3]
```diff
@@ -163,30 +163,42 @@ deployment target system:
...
-       scp LimeJeOS-Leap-42.3.x86_64-1.42.3.xz PXE_SERVER_IP:/srv/tftpboot/image/
-       scp LimeJeOS-Leap-42.3.x86_64-1.42.3.md5 PXE_SERVER_IP:/srv/tftpboot/image/
+       scp LimeJeOS-Leap-42.3.xz PXE_SERVER_IP:/srv/tftpboot/image/
+       scp LimeJeOS-Leap-42.3.md5 PXE_SERVER_IP:/srv/tftpboot/image/
+       scp LimeJeOS-Leap-42.3.initrd PXE_SERVER_IP:/srv/tftpboot/image/
+       scp LimeJeOS-Leap-42.3.kernel PXE_SERVER_IP:/srv/tftpboot/image/
 
 3. Copy the kernel command line parameters from
-   LimeJeOS-Leap-42.3.x86_64-1.42.3.append. Edit your PXE configuration
+   LimeJeOS-Leap-42.3.append. Edit your PXE configuration
```
Comment (tomschr) [inline doc/source/building/build_oem_disk.rst]: That LimeJeOS-Leap-42.3.append is a file, right? In that case use `:file:`.

File: doc/source/building/build_oem_disk.rst [hunk 4]
```diff
@@ -163,30 +163,42 @@ deployment target system:
...
-   system image and md5 file to the correct location on the server.
-   KIWI searches the image below SERVER_ROOT/image/
+   A typical PXE append line looks like the following example
+
+   .. code:: bash
+
+       append initrd=boot/initrd rd.kiwi.install.pxe rd.kiwi.install.image=tftp://192.168.100.16/image/LimeJeOS-Leap-42.3.xz
+
+   The location of the image is specified as a source URI which can point
+   to any location supported by the `curl` utility. KIWI calls `curl` to fetch
```
Comment (tomschr) [inline doc/source/building/build_oem_disk.rst]: Maybe use command instead of utility.

File: doc/source/building/build_oem_disk.rst [hunk 5]
```diff
@@ -163,30 +163,42 @@ deployment target system:
...
+   the data from this URI. This also means your image, md5 file, system kernel
+   and initrd could be fetched from any server and doesn't have to be stored
+   on the PXE_SERVER.
```
Comment (tomschr) [inline doc/source/building/build_oem_disk.rst]: Use backticks around PXE_SERVER (it's a placeholder, right?). Do the same in line 201.

File: doc/source/building/working_with_images/setup_pxe_bootserver.rst [hunk 6]
```diff
@@ -40,9 +41,10 @@ serve as an example. Depending on your network structure, the IP addresses,
 ranges and domain settings need to be adapted to allow the DHCP server to
 work within your network. If you already have a DHCP server running in your
 network, make sure that the filename and next-server are correctly set
```
Comment (tomschr) [inline doc/source/building/working_with_images/setup_pxe_bootserver.rst]: what are these `filename` and `next-server`? Are these options, parameters etc? In that case, I'd recommend to add the term "parameter" or "option" after these and enclose the terms in backticks.

File: doc/source/building/working_with_images/setup_pxe_bootserver.rst [hunk 7]
```diff
@@ -53,22 +55,36 @@ The following steps describe how to set up a new DHCP server instance:
 
    .. code:: bash
```
[the remark this answers is not in the document]
Comment (schaefi) [reply doc/source/building/working_with_images/setup_pxe_bootserver.rst]: yes makes sense did it that way

File: doc/source/building/working_with_images/setup_pxe_bootserver.rst [hunk 8]
```diff
@@ -53,22 +55,36 @@ The following steps describe how to set up a new DHCP server instance:
```
[the remark this answers is not in the document]
Comment (schaefi) [reply doc/source/building/working_with_images/setup_pxe_bootserver.rst]: Thanks I used this sentence, reads better than mine :)

**Comment by schaefi** [comment]
@tomschr Thanks for the review, how about it now ?

**Comment by tomschr** [comment]
@schaefi Looks good to me! 👍

**Comment by schaefi** [comment]
@tomschr so you give us your approval ;)

**Comment by tomschr** [comment]
@schaefi sure, always. 👍 😸

**Comment by schaefi** [comment]
ok, just the review state is still on "Changes requested" :)

Nevertheless I'll merge it now

Thanks much

**Pull request merged**